### ISSUE TITLE
requirements.txt package duplication

### ISSUE THREAD

**Issue by amttr** [bndr/pipreqs#65]
#### Ubuntu 16.10 x86_64. Python 2.7.13. pipreqs 0.4.6
```bash
$ mktmpenv -i pipreqs
$ git clone https://github.com/pallets/flask
$ pipreqs flask
$ cat flask/requirements.txt
```
> **setuptools==34.3.2**
> **setuptools==34.3.2**
> blinker==1.4
> click==6.7
> greenlet==0.4.12
> itsdangerous==0.24
> Jinja2==2.9.5
> Pygments==2.2.0
> pytest==3.0.7
> Sphinx==1.5.3
> Werkzeug==0.12.1

same with _--use-local_
> **setuptools==34.3.2**
> **setuptools==34.3.2**

**Comment by ryuzakyl** [comment]
I had the same issue.

In https://github.com/ryuzakyl/pipreqs/tree/hotfix/remove-package-duplicates, you can find a fix ;). If can, let me know if it really resolves this issue.

Already submitted a Pull Request.

**Comment by bndr** [comment]
This should be fixed with #67 from ryuzakyl. Tested with flask repository
